src/components/ApplicationForm/Skills.js

```diff
@@ -83,37 +83,71 @@ export default ({ errors, formInputs, onChange, role, handleResume }) => {
           </FormRow>
 
           {role === 'designer' ? (
-            <FormRow id="portfolio">
-              <TextInput
-                inline
-                placeholder="Optional"
-                size="large"
-                value={formInputs.portfolio}
-                invalid={!!errors.portfolio}
-                errorMsg={errors.portfolio}
-                onChange={e =>
-                  onChange({
-                    portfolio: e.target.value,
-                  })
-                }
-              />
-            </FormRow>
+            <>
+              <FormRow id="Personal website/portfolio link">
```

Comment:
For `id` values can we keep it to alphanumeric characters and use hyphens or underscores to break words? Don't think we're supposed to have whitespace in `id`

Docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
This id is also use to display the field value tho, I was actually considering renaming the property from id to something else like "fieldName". In particular the id field wasn't used in css styling